react-web/src/components/atoms/typography/link/styles.module.scss

```diff
@@ -0,0 +1,27 @@
+.link {
+  font-size: var(--font-size-lg);
+  padding: 8px 16px;
```

Comment:
Won't this padding end up causing problems if we add other texts? 🤔
I think it would be better to leave it without padding, what do you think?